x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/host_details.tsx

```diff
@@ -106,7 +106,7 @@ export const HostDetails = memo(({ details }: { details: HostMetadata }) => {
     path: agentDetailsWithFlyoutPath,
     state: {
       onDoneNavigateTo: [
-        'securitySolution:management',
+        'securitySolution:administration',
```

Comment:
This should actually be replaced with the constant we have in `management/common/constants`: `MANAGEMENT_APP_ID`